variables.tf

```diff
@@ -197,6 +197,12 @@ variable "redis_memory_size" {
   type        = number
 }
 
+variable "redis_version" {
+  type        = string
+  description = "The version of Redis to install"
+  default     = "REDIS_5_0"
```

Comment:
This was `REDIS_6_0` previously. Do we want to change to `REDIS_5_0`?

Reply:
I left it as the last working version. `REDIS_6_0` fails because of a bug, which is why I had to use the X. I hate to default to an `X`...

Reply:
Here's the [doc](https://cloud.google.com/memorystore/docs/redis/reference/rest/v1/projects.locations.instances). You can search for "redisVersion".

Reply:
Oh perfect! Thank you!